js/shared/record.js

```diff
@@ -188,13 +198,27 @@ window.ScratchpadRecord = Backbone.Model.extend({
             this.cacheRestore(-1 * this.seekCacheInterval);
         }
 
-        // Execute commands and build cache, bringing state up to current
-        for (var i = cacheOffset; i <= seekPos; i++) {
-            this.runCommand(this.commands[i]);
-            this.cache(i);
+        let currentOffset = cacheOffset;
```

Comment:
Could you add a comment above this section about why it uses RAF? (i.e. a short version of your PR descrip)

Reply:
For sure! Adding it now.